app/scripts/routes.js

```diff
@@ -11,6 +11,10 @@ define([], function() {
             '/login': {
                 templateUrl: '/views/login.html',
                 controller: 'LoginCtrl'
+            },
+            '/student/:docket': {
```

Comment:
This path is not OK. it should be `/students/:docket` ('s' from students missing)